scripts/azworker.sh

```diff
@@ -59,6 +59,9 @@ case ${1-} in
     rsync -az "$(dirname "${0}")/../build/disable-hyperv-timesync.sh" "${USER}@${FQDN}:bootstrap/"
     ssh -A "${USER}@${FQDN}" ./bootstrap/bootstrap-debian.sh
     ssh -A "${USER}@${FQDN}" ./bootstrap/disable-hyperv-timesync.sh
+    # tmux configuration (for persistent SSH): present a color terminal and disable tmux scrollback.
+    ssh -A "${USER}@${FQDN}" "echo 'set -g default-terminal screen-256color' > .tmux.conf"
```

Comment:
If `~/.tmux.conf` exists locally, it would be nice to copy it over instead of using this default. (but probably not worth another testing cycle just for this)